src/graphql/Hooks.graphql

```diff
@@ -61,6 +61,7 @@ type Hook {
   hookId: ID!
   metadata: HookMetadata!
   schedule: [String]!
+  bindings: JSON!
```

Comment:
Is there a particular format to a binding? Also, isn't bindings an _array_ of binding? I was thinking something more along the lines of:

```graphql
type HookBinding {
  # Define shape here
}

type Hook {
  # ...
  bindings: [HookBinding]!
  # ...
}
```